src/Backend/Core/Js/backend.js

```diff
@@ -1197,6 +1197,26 @@ jsBackend.forms = {
     jsBackend.forms.meta()
     jsBackend.forms.datePicker()
     jsBackend.forms.bootstrapTabFormValidation()
+    jsBackend.forms.imagePreview()
+  },
+
+  imagePreview: function () {
```

Comment:
@Senjutsuu can you add this to the frontend as well? That way it will also work for the image types there